app/models/Stage.java

```diff
@@ -11,25 +13,27 @@
 public class Stage {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    public Long id;
-    public int stageId;
-    public int raceId;
-    public String raceName;
-    public Date startTime;
-    public Date endTime;
-    public int distance;
-    public StageType stageType;
-    public String from;
-    public String to;
+    private Long id;
+    private int stageId;
+    private Date startTime;
+    private Date endTime;
+    private int distance;
+    private StageType stageType;
+    private String start;
+    private String destination;
 
     @ManyToOne(cascade=CascadeType.PERSIST)
-    public Race race;
+    @JsonBackReference
+    private Race race;
     @OneToMany(mappedBy="stage", cascade= CascadeType.ALL)
-    public List<RiderStageConnection> riderStageConnections = new ArrayList<RiderStageConnection>();
+    @JsonBackReference
+    private List<RiderStageConnection> riderStageConnections = new ArrayList<RiderStageConnection>();
     @OneToMany(mappedBy="stage", cascade= CascadeType.ALL)
-    public List<Maillot> mailllots = new ArrayList<Maillot>();
+    @JsonBackReference
+    private List<Maillot> mailllots = new ArrayList<Maillot>();
     @OneToMany(mappedBy="stage", cascade= CascadeType.ALL)
-    public List<Notification> notifications = new ArrayList<Notification>();
+    @JsonBackReference
+    private List<Notification> notifications = new ArrayList<Notification>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)